### PR TITLE
Release 10/03/23

### PR DESCRIPTION
**Pull request by sycombs**
The following packages will be released:

  - terra-clinical-data-grid@2.48.0
  - terra-clinical-item-view@4.13.0

### CODE REVIEW

File: packages/terra-clinical-data-grid/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 2.48.0 - (October 3, 2023)
+
+* Changed
+  * Minor dependency version bump
```
Comment (nikhitasharma) [inline packages/terra-clinical-data-grid/CHANGELOG.md:8]: What changes are we releasing here. I do not see any changes being released in the changelog other than the minor dependency bump changes on both packages.

Comment (sdadn) [reply packages/terra-clinical-data-grid/CHANGELOG.md:8]: This is automatically done by the release script where it bumps dependencies if it exists within the same monorepo. Version updates typically match dependency bumps.

Comment (sdadn) [reply packages/terra-clinical-data-grid/CHANGELOG.md:8]: Good question, there should be at least one package with a CHANGELOG entry that's not a dependency version bump

Comment (sycombs) [reply packages/terra-clinical-data-grid/CHANGELOG.md:8]: Good catch, it's the changes from this PR that are being released: https://github.com/cerner/terra-clinical/pull/913, looks like something went weird with the CHANGELOG during merge. Fixed with [5096889](https://github.com/cerner/terra-clinical/pull/921/commits/50968891f4cbb0e36f8f32adc6ff3154f12a9f15)

**Review by sdadn — APPROVED** [review]

**Pull request merged**